homeassistant/components/growatt_server/const.py

```diff
@@ -8,11 +8,15 @@
 DEFAULT_NAME = "Growatt"
 
 SERVER_URLS = [
-    "https://server.growatt.com/",
+    "https://server-api.growatt.com/",
     "https://server-us.growatt.com/",
     "http://server.smten.com/",
 ]
 
+DEPRECATED_URLS = [
+    "https://server.growatt.com/",
```

Comment:
Maybe change this to a keyvalue object so you can set the prefered fallback for the deprecated url

Reply:
I'd prefer not to if that's ok. 
I think we should try and steer everyone towards the `DEFAULT_URL` where possible.